src/styl/common.styl

```diff
@@ -125,7 +125,7 @@ span.fa
     margin-right 10px
   &:last-child
     border-bottom none
-  &.odd
+  &.even
```

Comment:
What's the goal of this? I'm not sure but it may have an impact on the rest of the screens.

Reply:
If you switch it, you'll see that it's hard to see where the background ends and the first item begins. By making the first item (item 0) contrast with the background, there is a clearer delineation of where the list starts. I just ran around the whole app and made sure this worked everywhere it was used.
I also tried to fix some stuff that looked obviously wrong as I was going. (e.g., the tabs in the analysis view were colored dark)